var/spack/repos/builtin/packages/gperftools/package.py

```diff
@@ -22,4 +22,19 @@ class Gperftools(AutotoolsPackage):
     version('2.4', sha256='982a37226eb42f40714e26b8076815d5ea677a422fb52ff8bfca3704d9c30a2d')
     version('2.3', sha256='093452ad45d639093c144b4ec732a3417e8ee1f3744f2b0f8d45c996223385ce')
 
-    depends_on("unwind")
+    variant('sized_delete', default=False, description="Build sized delete operator")
+    variant('dynamic_sized_delete_support', default=False, description="Try to build run-time switch for sized delete operator")
+    variant('debugalloc', default=True, description="Build versions of libs with debugalloc")
```

Comment:
@iarspider do you know if this has any runtime impact? Or is enabling `debugalloc` anyway the default, so this is in line with what was there before?

Reply:
As far as I can tell, it's on by default: https://github.com/gperftools/gperftools/blob/gperftools-2.9.1/configure.ac#L46